components/head.js

```diff
@@ -1,44 +0,0 @@
-import React from 'react'
```

Comment:
mmm why are you removing head.js here?

Reply:
This file is not used in our app. Should I add it back?